tests/unit/includes/BlockTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+class BlockTest extends WikiaDatabaseTest {
+	const BLOCKED_USER_ONE_ID = 174;
+	const BLOCKED_USER_ONE_NAME = 'BlockedUser';
+
+	const BLOCKED_USER_ONE_BLOCK_ID = 15;
+	const BLOCKED_USER_ONE_BLOCK_REASON = 'Vandalism';
+	const BLOCKED_USER_ONE_BLOCK_TS = '20110101000000';
+	const BLOCKED_USER_ONE_BLOCK_EXPIRY = 'infinity';
+
+	const BLOCKED_ANON_USER = '8.8.8.8';
+
+	const BLOCKED_ANON_USER_BLOCK_ID = 20;
+	const BLOCKED_ANON_USER_BLOCK_REASON = 'Spam';
+	const BLOCKED_ANON_USER_BLOCK_TS = '20130412050100';
+	const BLOCKED_ANON_USER_BLOCK_EXPIRY = '20360712050100';
+
+	const BLOCKING_ADMIN_ID = 192;
+	const BLOCKING_ADMIN_NAME = 'BlockingAdmin';
+
+	const NOT_BLOCKED_USER_ID = 283;
+	const NOT_BLOCKED_USER_NAME = 'NotBlockedUser';
+
+	const NOT_BLOCKED_ANON = '1.1.1.1';
+
+	private $wgMemc;
+
+	protected function setUp() {
+		parent::setUp();
+
+		$this->wgMemc = $GLOBALS['wgMemc'];
+		$GLOBALS['wgMemc'] = new EmptyBagOStuff();
+	}
```

Comment:
We have `WikiaDatabaseTest::disableMemCache` helper for, well, disabling memcache...

Reply:
That's only available in `WikiaBaseTest`, and trying to make `WikiaDatabaseTest` subclass `WikiaBaseTest` breaks `WikiaMockProxy` and associated hacks 😞